src/providers/mssql/qgsmssqlfeatureiterator.h

```diff
@@ -95,6 +95,9 @@ class QgsMssqlFeatureIterator : public QgsAbstractFeatureIteratorFromSource<QgsM
 
     bool prepareOrderBy( const QList<QgsFeatureRequest::OrderByClause> &orderBys ) override;
 
+    double ValidLat( const double latitude );
```

Comment:
```suggestion
    double validLat( const double latitude );
```

Members should start lower case.

Reply:
Applied the lower case style for those functions which are affected by this change. There are some other unrelated functions which are also upper case. That should probably be handled by a separate ticket.